x-pack/elastic-agent/.gitignore

```diff
@@ -1,5 +1,6 @@
 # agent
 build/
+elastic-agent
```

Comment:
Should we ignore `*.exe` as well?

Reply:
Good point, but it seems to be enough with the `*.exe` in the main `.gitignore`. https://github.com/elastic/beats/blob/17615155c11cdc4ec6e4e2097d69c328da329aa8/.gitignore#L44